ietf/templates/meeting/proceedings.html

```diff
@@ -173,11 +173,39 @@ <h2 class="mt-5" id="irtf">
                 </tr>
             </thead>
             <tbody>
-                {% for entry in irtf %}
+                {% for entry in irtf.meeting_groups %}
                     {% include "meeting/group_proceedings.html" with entry=entry meeting=meeting show_agenda=True only %}
                 {% endfor %}
             </tbody>
         </table>
+        {% if irtf.not_meeting_groups %}
+        <p>
+            IRTF groups not meeting:
+            {% for entry in irtf.not_meeting_groups %}
+                {% if entry.name == "" %}{# do not show named sessions in this list #}
+                    <a href="{% url 'ietf.group.views.group_home' acronym=entry.group.acronym %}">
+                        {{ entry.group.acronym }}
+                    </a>{% if not forloop.last %},{% endif %}
+                {% endif %}
+            {% endfor %}
+        </p>
+        <table class="table table-sm table-striped">
+            <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for entry in irtf.not_meeting %}{% if entry.sessions_with_materials %}
```

Comment:
This needs to be updated to match the corresponding change in #5065. (That is, it should check `entry.has_materials` instead)

Reply:
hoop - old copy of main :(

Reply:
{"id_str":"429799239393476608","retweet_count":0,"display_text_range":[0,109],"is_quote_status":false,"source":"<a href=\"http:\/\/www.apple.com\" rel=\"nofollow\">iOS<\/a>","lang":"en","place":null,"full_text":"Check out my classic car in #CSRClassics for iPhone! It's FREE! http:\/\/t.co\/ttSFt3gn6K http:\/\/t.co\/ZYm0vHNdZR","favorite_count":0,"user":{"profile_sidebar_border_color":"C0DEED","favourites_count":54,"geo_enabled":true,"profile_text_color":"333333","listed_count":2,"profile_background_image_url_https":"https:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","is_translation_enabled":false,"verified":false,"follow_request_sent":false,"notifications":false,"profile_background_image_url":"http:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","screen_name":"AkaChiefy","withheld_in_countries":[],"profile_banner_url":"https:\/\/pbs.twimg.com\/profile_banners\/2319153741\/1640789674","utc_offset":null,"lang":null,"entities":{"url":{"urls":[{"display_url":"github.com\/website","indices":[0,23],"url":"https:\/\/t.co\/wPTpJaJxqb","expanded_url":"http:\/\/github.com\/website"}]},"description":{"urls":[]}},"url":"https:\/\/t.co\/wPTpJaJxqb","friends_count":427,"followers_count":49,"profile_sidebar_fill_color":"DDEEF6","translator_type":"none","profile_use_background_image":true,"statuses_count":1295,"created_at":"Sun Feb 02 02:03:19 +0000 2014","profile_background_color":"C0DEED","default_profile_image":false,"contributors_enabled":false,"has_extended_profile":true,"location":"Phoenix, AZ","time_zone":null,"profile_image_url":"http:\/\/pbs.twimg.com\/profile_images\/1476205004205690881\/n3SRc7qU_normal.jpg","id_str":"2319153741","is_translator":false,"profile_link_color":"1DA1F2","profile_background_tile":false,"id":2319153741,"protected":false,"default_profile":true,"description":"Fun loving dog dad of two amazing Australian Cattle Dogs - Chief the Blue Heeler and Koda the Red Heeler. perpetual energy","profile_image_url_https":"https:\/\/pbs.twimg.com\/profile_images\/1476205004205690881\/n3SRc7qU_normal.jpg","following":false,"name":"Todd Gifford"},"in_reply_to_user_id":null,"coordinates":null,"in_reply_to_status_id":null,"geo":null,"id":4.2979923939347661e+17,"in_reply_to_user_id_str":null,"extended_entities":{"media":[{"expanded_url":"https:\/\/twitter.com\/tgifford37\/status\/429799239393476608\/photo\/1","indices":[87,109],"id":4.2979923921314611e+17,"url":"http:\/\/t.co\/ZYm0vHNdZR","id_str":"429799239213146113","media_url_https":"https:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","media_url":"http:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","display_url":"pic.twitter.com\/ZYm0vHNdZR","type":"photo","sizes":{"thumb":{"h":150,"resize":"crop","w":150},"medium":{"h":640,"resize":"fit","w":1136},"small":{"h":383,"resize":"fit","w":680},"large":{"h":640,"resize":"fit","w":1136}}}]},"entities":{"media":[{"expanded_url":"https:\/\/twitter.com\/tgifford37\/status\/429799239393476608\/photo\/1","indices":[87,109],"id":4.2979923921314611e+17,"url":"http:\/\/t.co\/ZYm0vHNdZR","id_str":"429799239213146113","media_url_https":"https:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","media_url":"http:\/\/pbs.twimg.com\/media\/Bfb0IYIIYAEFPBy.jpg","display_url":"pic.twitter.com\/ZYm0vHNdZR","type":"photo","sizes":{"medium":{"h":640,"resize":"fit","w":1136},"large":{"h":640,"resize":"fit","w":1136},"thumb":{"h":150,"resize":"crop","w":150},"small":{"h":383,"resize":"fit","w":680}}}],"hashtags":[{"indices":[28,40],"text":"CSRClassics"}],"urls":[{"indices":[64,86],"url":"http:\/\/t.co\/ttSFt3gn6K","expanded_url":"http:\/\/nmgam.es\/cct","display_url":"nmgam.es\/cct"}],"symbols":[],"user_mentions":[]},"truncated":false,"contributors":null,"in_reply_to_status_id_str":null,"favorited":false,"retweeted":false,"in_reply_to_screen_name":null,"possibly_sensitive":false,"created_at":"Sun Feb 02 02:11:38 +0000 2014"}